### PR TITLE
Add adoption list queries

### LINKED ISSUE #74: Create all adoptions and my adoptions query
## Overview
- A query that returns all adoption options closest to the user.
- A query that returns all adoptions the user has posted.

#30

### PR DESCRIPTION
**Pull request by belgamo**
closes #74 

#### Why I decided to use `forwardConnectionArgs` instead `connectionArgs`?
Because in the app's home screen slider we can't go back to a previous adoption as shown in [this PR](https://github.com/animavita/animavita/pull/66).

#### CHANGELOG
- Added adoption graphql types and model
- Added missing .editorconfig file
- Updated GraphQL schema

The improved version of [this](https://github.com/animavita/animavita/pull/58).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
```
Comment (andrerpbts) [inline .editorconfig]: Extra line

File: packages/graphql/src/modules/adoption/AdoptionQuery.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import AdoptModel from './AdoptionModel';
+import AdoptionType from './AdoptionType';
+
+export const AdoptionConnection = connectionDefinitions({
+  name: 'Adoption',
+  nodeType: AdoptionType,
+});
+
+export const AdoptionsQueries: GraphQLFieldConfigMap<any, GraphQLContext, any> = {
```
Comment (renanmav) [inline packages/graphql/src/modules/adoption/AdoptionQuery.ts:14]: Why not specifying the first generic type of `GraphQLFieldConfigMap`?

**Review by renanmav — COMMENTED** [review]
Nice job!

**Comment by petruspierre** [comment]
LGTM 👏

File: packages/graphql/src/modules/adoption/AdoptionModel.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import mongoose, {Document} from 'mongoose';
+
+const {ObjectId} = mongoose.Types;
+
+type AnimalGenders = 'male' | 'female';
```
Comment (brunohkbx) [inline packages/graphql/src/modules/adoption/AdoptionModel.ts]: Enum works here: https://www.typescriptlang.org/docs/handbook/enums.html

File: packages/graphql/src/modules/adoption/AdoptionQuery.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  adoptions: {
+    type: AdoptionConnection.connectionType,
+    args: forwardConnectionArgs,
+    resolve: (_, args) => connectionFromPromisedArray(AdoptModel.find().populate('user') as any, args),
```
Comment (brunohkbx) [inline packages/graphql/src/modules/adoption/AdoptionQuery.ts]: IMO I think it's better to move these resolvers to another file. They can get big over time and will be hard to read having them inline.

File: .editorconfig (hunk 1 above)
Comment (belgamo) [reply .editorconfig]: Linter default config...

**Pull request merged**